### ISSUE TITLE
Make options settable at runtime

### ISSUE THREAD

**Issue by bneumann** [opensheetmusicdisplay/opensheetmusicdisplay#407]
Currently the options are passed when creating a new instance. It would be good to set the options also after creation, so only a re-render is necessary to update the view.

Background: While fiddeling with an react component integration I figured that passing options and altering the components state results in a second render view. This is due to the nature of react and I haven't found an easy way around. But I thought it would be a good idea to have it in OSMD anyways.

Example use case (Haven't tried the code, just from memory):
```javascript
var options = {drawTitle: true};
var osmd = undefined;
osmd = new OpenSheetMusicDisplay(fileName, options).then(() => osmd.render());
// Now change settings
options.drawTitle = false;
// 1. soultions: Do implicit render:
osmd.applyOptions(options);
// 2. solution: Change options and render
osmd.Options = options;
osmd.render();
```

**Comment by sschmidTU** [comment]
Absolutely, I wanted to have a public method to set every setting at runtime anyways and the constructor only calls that method using the options object.
Since i created the options class, i could do this as well.

**Comment by bneumann** [comment]
Thanks @sschmidTU

**Comment by sschmidTU** [comment]
Options are now settable at runtime, same method as in the constructor, renamed to setOptions().
Small fix in 77d9596f14ae80588619f308887899f1540bc85f.

The only option that can't be set during runtime yet is changing the backend.
Something about clearing the previous canvas and setting a new canvas to render on doesn't work yet.

**Comment by bneumann** [comment]
Nice! I'll check it out
